CodeStarterCamp_Week2/File.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 이선준 on 2023/03/10.
+//
+
+//MARK: STEP2
+
+import Foundation
+
+func createWinningLottoNumbers() -> Set<Int> {
+    var lottoNumberSet: Set<Int> = Set<Int>()
+    while lottoNumberSet.count < 6 {
+        lottoNumberSet.insert(Int.random(in: 1...45))
+    }
+    return lottoNumberSet
+}
+
+func compareLottoNumber(with myLottoNumbers: [Int], to winningLottoNumbers: Set<Int>) {
```

Comment:
네이밍 고민하신 부분을 잘 설명해주셔서 좋았습니다.
좋은 고민을 하셨다 생각해요! 👏

저도 영알못이긴 하지만 😅 약간 응용을 해보자면 이렇게 쓸 수도 있을거 같아요
```suggestion
func compare(_ myLottoNumbers: [Int], with winningLottoNumbers: Set<Int>) {
```

Reply:
호출부에서는 이런식으로 호출되겠네요!
```swift
compare(myLottoNumbers, with: winningLottoNumbers)
```